daml-assistant/get-daml.sh

```diff
@@ -10,7 +10,20 @@
 # For more information please visit https://daml.com/ and https://docs.daml.com/
 #
 
+#
+# USAGE:
+#    get-daml.sh            Download and install the latest DAML SDK release.
+#    get-daml.sh VERSION    Download and install given version of DAML SDK.
+#
+
+ARG1="$1"
```

Comment:
I would just be explicit and use `ARG1="${1:-}" instead (or do that later when you reference it).

Reply:
Thanks! :+1: